conda_lock.py

```diff
@@ -16,31 +16,33 @@
 import yaml
 import os
 import requests
+import tempfile
+
 
 DEFAULT_PLATFORMS = ["osx-64", "linux-64", "win-64"]
-FAKE_PREFIX_NAME = "__magicmarker"
-FAKE_PKGS_ROOT = "/tmp/something/that/does/not/exist"
 
 
 def solve_specs_for_arch(channels, specs, platform):
     # type: (typing.List[str], typing.List[str], str) -> dict
-    args = [
-        "conda",
-        "create",
-        "-p",
-        FAKE_PREFIX_NAME,
-        "--override-channels",
-        "--dry-run",
-        "--json",
-    ]
-    for channel in channels:
-        args.extend(["-c", channel])
-    args.extend(specs)
 
     env = dict(os.environ)
-    env.update({"CONDA_SUBDIR": platform, "CONDA_PKGS_DIRS": FAKE_PKGS_ROOT})
```

Comment:
Changed the names of the variables to match the conda equivalent for readability. I also expanded all the short optional args for the same reason.